CONTRIBUTING.md

```diff
@@ -1,177 +1,141 @@
 # Contributing to Operation Code
 
-Looking to contribute something to Operation Code? **Here's how you can help.**
+So you want to learn how to program? Contributing to Operation Code is a great place to get started! This document will help you get from 0 to deploying code in no time! 
 
-Please take a moment to review this document in order to make the contribution process seamless and a joy for everyone involved.
+## Table Of Contents
+1. [Quickstart](#quickstart)
+2. [Finding an issue](#finding-an-issue)
+3. [Setting up your environment](#setting-up-your-environment)
+4. [Working On Your Issue](#getting-the-code)
+5. [Submitting your changes](#submitting-your-changes)
```

Comment:
Should be `Working on your issue` w/o capitalizing every word - to be consistent with the other quickstart headings